Code/GraphMol/RGroupDecomposition/RGroupCore.cpp

```diff
@@ -816,4 +817,26 @@ int RCore::matchingIndexToCoreIndex(int matchingIndex) const {
                   "Matched atom missing core index");
   return atom->getProp<int>(RLABEL_CORE_INDEX);
 }
+
+// Create tautomer query for the matching mol on demand and cache for performance
```

Comment:
Could we also indicate here that when called, if a tautomerQuery is returned, then this will automatically use it and if not the normal match will be performed.   This will document the confusion I had below.

Reply:
I just have a document suggestion, otherwise LGTM!  Can't wait to use it.

Reply:
I've added code documentation round Brian's issue.